actions/docs-verifier/src/RedirectionVerifier/RedirectionHelpers.cs

```diff
@@ -7,39 +7,27 @@ namespace RedirectionVerifier
 {
     public static class RedirectionHelpers
     {
-        public static async Task<ImmutableArray<Docset>> GetDocsetsAsync()
+        public static async Task<ImmutableArray<string>?> GetRedirectionFilesAsync()
         {
             OpenPublishingConfigReader configReader = new();
             return await configReader.MapConfigurationAsync();
         }
 
         public static async Task<ImmutableArray<string>> GetRedirectionFileNames()
         {
-            ImmutableArray<Docset> docsets = await GetDocsetsAsync();
-
-            if (docsets.IsDefault)
-                return default;
-
-            var redirectionFileNames = new List<string>();
-
-            // If there's more than one docset, combine all the redirection file names.
-            foreach (Docset docset in docsets)
-            {
-                if (docset.RedirectionFiles != null)
-                    redirectionFileNames.AddRange(docset.RedirectionFiles);
-            }
+            ImmutableArray<string>? redirectionFileNames = await GetRedirectionFilesAsync();
 
             // If no redirection files are found in the OPS config, just use the default name.
-            if (redirectionFileNames.Count == 0)
-                redirectionFileNames.Add(".openpublishing.redirection.json");
+            if (redirectionFileNames == null)
+                redirectionFileNames = new ImmutableArray<string>() { ".openpublishing.redirection.json" };
```

Comment:
Minor simplification. No behavioral change.

```suggestion
                redirectionFileNames = ImmutableArray.Create(".openpublishing.redirection.json");
```